docs/resources/zapi-and-rest-gap.md

```diff
@@ -0,0 +1,7 @@
+# Zapi and Rest gaps
+
+## Volume Count difference
+`REST` and `ZAPI` collector will return different counts of volume object depending on whether you have object store servers setup on your cluster or not.
+
```

Comment:
The `REST` and `ZAPI` collectors return a different number of `volume_label`s depending on whether you have set up object store servers on your cluster.